src/utils.ts

```diff
@@ -12,25 +11,6 @@ export function exec(cmd: string, args: readonly string[]): Promise<{ stdout: st
 	});
 }
 
-export function httpsRequest(url: string, options: https.RequestOptions = {}, encoding = 'utf8'): Promise<string> {
```

Comment:
Was that old unused code?

Reply:
Yep, see the commit message - I pushed it intentionally in a separate one 😉

Reply:
Ah, I did not look at the commits individually 😅